### ISSUE TITLE
question about environment light source image  L

### ISSUE THREAD

**Issue by wang21jun** [lllyasviel/IC-Light#107]
Thanks for your great work!
Why the environment light source image L, mentioned as a prompt embedding in your paper, is not included in your infer code?

**Comment by YeJiangnan1029** [comment]
Same question. The paper mentions that L is encoded as 3 768-dim tokens, which is the same as the shape of text embedding. Does the light embedding (3 tokens) be removed in the inference stage? Which tokens have been replaced with(or added to) the light embeddings among 77 text tokens in the trainning stage? Or is it posible that make use of the light embeddings(or environment light L) to control the result of inference result? Waiting for your kind reply! :)

**Comment by zyx1213271098** [comment]
same question

**Comment by laolongboy** [comment]
same question

**Comment by bhosalems** [comment]
Without training code we will not know how the model is trained.

**Comment by jxl0131** [comment]
> Without training code we will not know how the model is trained.

But if you use L in training, you must use L in inferring

**Comment by pljj315** [comment]
> > Without training code we will not know how the model is trained.
> 
> But if you use L in training, you must use L in inferring

maybe L is not working well after training? means using L or not using L results in the very same results...but it's strange since L should work finely to have the influence with "consistency loss"
